modules/core/include/visp3/core/vpImage.h

```diff
@@ -185,6 +185,8 @@ template <class Type> class vpImage
   Type getMinValue() const;
   // Look for the minumum and the maximum value within the bitmap
   void getMinMaxValue(Type &min, Type &max) const;
+  // Look for the minumum and the maximum value within the bitmap and get their location
+  void getMinMaxLoc(vpImagePoint *minLoc, vpImagePoint *maxLoc, Type *minVal=NULL, Type *maxVal=NULL) const;
```

Comment:
I would prefer here parameters passed as references rather than pointers

Reply:
I would like to keep it like this. This allows:
`I.getMinMaxLoc(&min_loc, NULL, &min_val, NULL);`
as usually we want to retrieve only the min/max value and the min/max location.

Reply:
ok. But then please add a sample code in the doc of this function with an example to get all the values, and an other one to retrieve only min & max